src/app/actions/createMembership.tsx

```diff
@@ -118,6 +121,17 @@ export async function createMembership(
     },
   })
 
+  // Send confirmation email
+  const confirmationEmail = render(<MembershipConfirmationEmail />)
```

Comment:
Do we really want to render here? Could we use other methods to get the email html?

Reply:
This is how the documentation is showing it and it seems to be good enough. I guess we could potentially render it beforehand and export it. Worth a try?

Reply:
Alright, yeah, maybe an export before hand would be great I believe